apps/discovery_streams/mix.exs

```diff
@@ -56,35 +58,15 @@ defmodule DiscoveryStreams.Mixfile do
       {:placebo, "~> 1.2", only: [:dev, :test, :integration]},
       {:redix, "~> 0.10.2"},
       {:sweet_xml, "~> 0.6"},
-      {:smart_city, "~> 2.8"},
-      {:smart_city_test, "~> 0.5", only: [:test, :integration]},
+      {:smart_city, "~> 3.0"},
+      {:smart_city_test, "~> 0.8", only: [:test, :integration]},
       {:streaming_metrics, "~>2.1"},
       {:temporary_env, "~> 2.0", only: [:test, :integration]},
       {:sobelow, "~> 0.8", only: :dev, runtime: false},
-      {:husky, "~> 1.0", only: :dev, runtime: false},
-      # updating version breaks
-      {:distillery, "2.0.14"},
-      # distillery breaks @ 2.1.0 due to elixir 1.9 support
-      {:poison, "3.1.0"}
+      {:distillery, "~> 2.1"},
       # poison breaks @ 4.0.1 due to encode_to_iotdata missing from 4.0
-    ]
-  end
-
-  defp docs do
-    [
-      main: "readme",
-      source_url: "https://github.com/smartcitiesdata/discovery-streams",
-      extras: [
-        "README.md"
-      ]
-    ]
-  end
-
-  defp package do
-    [
-      maintainers: ["smartcitiesdata"],
-      licenses: ["Apache 2.0"],
-      links: %{"GitHub" => "https://github.com/smartcitiesdata/discovery-streams"}
+      # {:poison, "~> 4.0"},
```

Comment:
Can we axe this comment?

Reply:
It's kind of a #TODO - we'll want to upgrade at some point to keep from sticking with Poison 3.x forever, but it's beyond the scope of this card.

Reply:
Removed the commented out dep.

Reply:
DONE